omega/linear_regression/test_linear_regression.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
```

Comment:
This file belongs to `omega/tests` being outside the test folder the tests in this file will not be executed.